index.html

```diff
@@ -0,0 +1 @@
+<html> <body> hello <div> the </div> </body> </html>
```

Comment:
Please Format your code!

Reply:
ok

Reply:
@khaledkz
we made the modification kindly check

Reply:
ok